### ISSUE TITLE
PEP518: specification changes history

### ISSUE THREAD

**Issue by stanislavlevin** [pypa/packaging.python.org#1645]
Is https://peps.python.org/pep-0518 specification superseded by https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-build-system-dependencies-the-build-system-table ?

If so, there is no history of changes, e.g. removal of `wheel` from default `PEP518` dependencies.

**Comment by sinoroc** [comment]
> Is https://peps.python.org/pep-0518 specification superseded by https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-build-system-dependencies-the-build-system-table ?

Yes. Once accepted, the content of a PEP is migrated to an actual specification document, and the PEP is nothing more than an historical document.

> If so, there is no history of changes, e.g. removal of `wheel` from default `PEP518` dependencies.

If I recall correctly it is because the process I described above, was not really in place at the time of PEP518's acceptance (or at least the process was not followed scrupulously). So there was a long time between the PEP's acceptance and the writing of the specification document. Probably during that time the presence of `wheel` was made unnecessary.
